index.html

```diff
@@ -159,10 +172,13 @@ <h3>Multi-Post Stories <span>Gain + Glory</span></h3>
 
         <section class="about-me">
             <div class="about">
-                <h1 class="intro">About me</h1>
+                <h1 class="intro one">About me</h1>
```

Comment:
- [ ] Kindly ensure that the highlighted text stretches across to match the design

- Your design
<img width="908" alt="image" src="https://github.com/Wils-FOURR-speed/My-portfolio/assets/41163571/07383151-caf8-4e00-9dec-47dc45ac2b87">

- Figma design
<img width="405" alt="image" src="https://github.com/Wils-FOURR-speed/My-portfolio/assets/41163571/a2ca6901-074c-4062-b545-f651335c6cff">

Reply:
Please i've tried to do this but it seems not working